### DISCUSSION TITLE
OAuth2 support

### DISCUSSION THREAD

**Discussion by burkaslarry** [serverpod/serverpod#2443] [Ideas]
I would love to see if there are some guys working on OAuth support for server pods, which can greatly enhance the authentication  and encourage the teammate to go full in Dart instead of JAVA. 

https://spring.io/guides/tutorials/spring-boot-oauth2

OAuth 2.0 Documentation
https://oauth.net/2/

**Comment by SandPod** [comment]
We will add a RFC for this soon. We would like to add this for the future.
